src/NuGet/Microsoft.Net.Compilers.Toolset/AnyCpu/Microsoft.Net.Compilers.Toolset.Package.csproj

```diff
@@ -38,18 +39,18 @@
                       Targets="Publish"
                       ReferenceOutputAssembly="false"
                       SkipGetTargetFrameworkProperties="true"
-                      Condition="'$(TargetFramework)' == 'net6.0'"
-                      SetTargetFramework="TargetFramework=net6.0" />
+                      Condition="'$(TargetFramework)' == 'net7.0'"
+                      SetTargetFramework="TargetFramework=net7.0" />
   </ItemGroup>
 
   <Target Name="_GetFilesToPackage" DependsOnTargets="$(_DependsOn)">
     <ItemGroup>
       <_File Include="@(DesktopCompilerArtifact)" TargetDir="tasks/net472"/>
       <_File Include="@(DesktopCompilerResourceArtifact)" TargetDir="tasks/net472"/>
-      <_File Include="@(CoreClrCompilerBuildArtifact)" TargetDir="tasks/net6.0"/>
-      <_File Include="@(CoreClrCompilerToolsArtifact)" TargetDir="tasks/net6.0"/>
-      <_File Include="@(CoreClrCompilerBinArtifact)" TargetDir="tasks/net6.0/bincore"/>
-      <_File Include="@(CoreClrCompilerBinRuntimesArtifact)" TargetDir="tasks/net6.0/bincore/runtimes"/>
+      <_File Include="@(CoreClrCompilerBuildArtifact)" TargetDir="tasks/net7.0"/>
+      <_File Include="@(CoreClrCompilerToolsArtifact)" TargetDir="tasks/net7.0"/>
+      <_File Include="@(CoreClrCompilerBinArtifact)" TargetDir="tasks/net7.0/bincore"/>
+      <_File Include="@(CoreClrCompilerBinRuntimesArtifact)" TargetDir="tasks/net7.0/bincore/runtimes"/>
```

Comment:
I think I have found ways to condition most all of these TFM changes except these `Microsoft.Net.Compilers.Toolset.Package` projects.

We could define a duplicate set of ItemGroups and PropertyGroups in this file for source-build and for not, but it feels wrong to repeat these. Would it work to conditionally define a property like `NetTFM` at the top of the project and let it be interpreted in these paths?

Reply:
I'm not sure we can do this yet. I need to think about it for a bit.